Fix readmes and docs content (#866)
epinio/epinio: docs/howtos/development.md

### Installing Epinio

> You can have a look at [the dedicated document](https://docs.epinio.io/installation/installation.html) for cluster

Reviewer: ```suggestion
While we have a [dedicated document](https://docs.epinio.io/installation/installation.html) for cluster
```

> specific instructions, but generally this should be sufficient to get you running:

Reviewer: ```suggestion
specific instructions, the following should generally be sufficient to get you running:
```
